gt-profile.yml

```diff
@@ -0,0 +1,106 @@
+BagIt-Profile-Info:
+  BagIt-Profile-Identifier: https://ocr-d.github.io/gt-profile.json
+  BagIt-Profile-Version: '1.2.0'
+  Source-Organization: OCR-D
+  External-Description: BagIt profile for OCR line Ground Truth
+  Contact-Name: Konstantin Baierer
+  Contact-Email: konstantin.baierer@sbb.spk-berlin.de
+  Version: 0.1
+Bag-Info:
+  Bagging-Date:
+    required: false
+  Source-Organization:
+    required: false
+  Gt-Transcription-Extension:
+    required: false
+    default: '.gt.txt'
+  Gt-Transcription-Media-Type:
+    required: false
+    default: 'text/plain'
+  Gt-Transcription-Directory:
+    required: false
+    default: 'text'
+  Gt-Transcription-Normalization:
+    required: false
+    default: 'NFKC'
+    values:
+      - NFD
+      - NFKD
+      - NFC
+      - NFKC
+  Gt-Color-Image-Extension:
+    required: false
+    default: '.color.png'
+  Gt-Color-Image-Media-Type:
+    required: false
+    default: 'image/png'
+    values:
+      - 'image/png'
+      - 'image/tiff'
+      - 'image/jpeg'
+  Gt-Color-Image-Directory:
+    required: false
+    default: 'img'
+  Gt-Grayscale-Image-Extension:
+    required: false
+    default: '.nrm.png'
+  Gt-Grayscale-Image-Media-Type:
+    required: false
+    default: 'image/png'
+    values:
+      - 'image/png'
+      - 'image/tiff'
+      - 'image/jpeg'
+  Gt-Grayscale-Image-Directory:
+    required: false
+    default: 'grayscale'
+  Gt-Bitonal-Image-Extension:
+    required: false
+    default: '.bin.png'
+  Gt-Bitonal-Image-Media-Type:
+    required: false
+    default: 'image/png'
+    values:
+      - 'image/png'
+      - 'image/tiff'
+      - 'image/jpeg'
```

Comment:
JPEGs don't work for bitonal images as they are interpolated and always contain intermediate values. There might be some way to re-binarize them "losslessly" but it isn't straightforward.